corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name.capitalize
```

Comment:
- [ ] in order to change the variable you need to use the exclamation point after the `capitalize` method, so it will look something like this `name.capitalize!`

Reply:
Thanks, @mikethreels. fixed